README.md

```diff
@@ -32,13 +32,6 @@ Before proceeding, make sure that there are no current builds in the `Build` ste
 
-9. (Temporary for now). Create a branch off of the `prod branch` of the [openliberty.io repo](https://github.com/OpenLiberty/openliberty.io). Make the following updates all in the same branch and merge them into `draft` and `staging`:
-  
-   * Update the version of `{replace_version page.url 'X.0.0.X' }` in [src/main/content/antora_ui/src/partials/head-info.hbs](https://github.com/OpenLiberty/openliberty.io/blob/staging/src/main/content/antora_ui/src/partials/head-info.hbs) so that the canonical meta tag is set to the latest version of the documentation for all versions of the docs. 
-   * Update the version of the url `<loc>https://openliberty.io/docs/X.0.0.X/overview.html</loc>` in the [src/main/content/sitemap.xml](https://github.com/OpenLiberty/openliberty.io/blob/staging/src/main/content/sitemap.xml).
-
-10. Before proceeding, make sure all of the pull requests and changes from previous steps have been merged into their respective repos. Make a pull request from your branch into both the `draft` and `staging` branches. Once the pull requests have been reviewed merge them in. After each pull request has been merged in, the [draft site](https://draft-openlibertyio.mybluemix.net/) and [staging site](https://staging-openlibertyio.mybluemix.net/) should automatically rebuild and redeploy (takes ~60 minutes). If you have access to the IBM Cloud pipelines, you can track the progress through [IBM Cloud draft pipeline](https://cloud.ibm.com/devops/pipelines/3f4abe1b-0c51-4752-ac48-5ed9030f530f?env_id=ibm:yp:us-south) and [IBM Cloud staging pipeline](https://cloud.ibm.com/devops/pipelines/61002d76-1c13-4502-a109-3b794b3f2350?env_id=ibm:yp:us-south). Test that the docs link works and redirects to the newest doc version once the site is done building.
-
-11. In the openliberty.io repo, [open a pull request from the `staging` branch into the `prod` branch](https://github.com/OpenLiberty/openliberty.io/compare/staging?expand=1). The [openliberty.io site](https://openliberty.io/) will automatically rebuild when the pull request is merged in. You can track the progress using the [IBM Cloud pipeline](https://cloud.ibm.com/devops/pipelines/063d397c-febc-4f73-8340-61da6bc775f5?env_id=ibm:yp:us-south).
+9. In the openliberty.io repo, [open a pull request from the `staging` branch into the `prod` branch](https://github.com/OpenLiberty/openliberty.io/compare/staging?expand=1). The [openliberty.io site](https://openliberty.io/) will automatically rebuild when the pull request is merged in. You can track the progress using the [IBM Cloud pipeline](https://cloud.ibm.com/devops/pipelines/063d397c-febc-4f73-8340-61da6bc775f5?env_id=ibm:yp:us-south).
```

Comment:
I'm not sure I follow as to why steps 8 & 9 still exist.  If the docs team no longer needs to make changes to the openliberty.io repo, then they should no longer need to open PRs to it.  The draft and staging sites will be automatically rebuilt by their necessary merges to the docs repos (outlined in the previous steps).

The only potential addition would be to request a rebuild of the prod site, which could also be automated based on merges to `prod` branch of docs-playbook, similarly to how merges to `prod` branch of openliberty.io and blogs repos kicks off a prod site rebuild.  Alternatively, they could just request one via slack, though I'd be in favor of automation.

Reply:
@dmuelle @kinueng

Reply:
yeah ok @mbroz2 I will make that change

Reply:
removed

Reply:
This LGTM if we dont need to worry about changes in the website repo or the docs redirects anymore.

I think we need to add a final step to request a rebuild of the website until the automation piece gets worked out.

 If we do set it up so that a merge into Docs playbook prod triggers a prod site rebuild, we might need to revisit these instructions as the last merge to docs playbook prod is currently in step 4- not sure if it matters that we'd trigger a site rebuild before we're finished updating draft and staging stuff in the docs and docs generated repos.

Reply:
Hi @mbroz2, can we mark this code review feedback as resolved?

Reply:
After reading the latest changes to the instructions, this looks to be resolved.

Reply:
After reading the latest changes to the instructions, this looks to be resolved.